src/Libraries/CoreNodeModels/String.cs

```diff
@@ -104,8 +107,6 @@ public FormattedStringFromObject() : base("__ToStringFromObjectAndFormat")
     [OutPortTypes("string")]
     [IsDesignScriptCompatible]
     [AlsoKnownAs("DSCoreNodesUI.StringNodes.FromObject", "DSCoreNodesUI.FromObject")]
-    [Obsolete("this node is obsolete, please use the version of string from _ with numeric format option")]
-    [NodeDeprecated]
```

Comment:
My reasoning was as follows:

Since I am marking the new nodes experimental, I did not want to also mark the old nodes as obsolete (hiding them from the lib) - then it would not be clear to users how to proceed in a safe / future proof way.

I'd like to propose we follow this pattern when introducing new nodes (that replace existing ones):

phase 1: 
* Introduce new nodes experimental.
* Old nodes are untouched.
phase 2:
* gather feedback on experimental nodes.
* experimental nodes can be changed at any time.
phase 3:
* experimental attribute is removed from new nodes.
* old nodes are marked obsolete
phase 4:
* old nodes are removed in a major release.